src/api/d2-api.ts

```diff
@@ -33,6 +35,7 @@ export class D2ApiDefault {
         this.connection = prepareConnection(this.apiPath, auth);
         this.metadata = new D2ApiMetadata(this);
         this.currentUser = new D2ApiCurrentUser(this);
+        this.analytics = new Analytics(this);
```

Comment:
Not sure about the naming: D2ApiMetadata, D2ApiCurrentUser, Analytics

Reply:
Yep, we should be consistent. Not a big deal since it's unexposed code, but still. Do we remove the d2 prefix?

Reply:
Yeah, I believe it's too redundant. Analytics, Metadata, CurrentUser LGTM.